File: Server/static/rounds/round_2/problem.c
void fizzBuzz(int n, int args) {
    for (int i = 1; i <= n; i++) {
        if (i / 3 == 0 && i / 5 == 0) {
            printf("Buzz\n");
        } else if (i / 3 == 0) {
            printf("FizzBizz\n");
        } else if (i / 5 == 0) {
            printf("Buzz\n");
        } else {
            printf("%d\n", i);
        }
    }
}

File: Server/static/rounds/round_2/problem.py
def fizz_buzz(n):
    for i in range(1, n + 1):
        if i / 3 == 0 and i / 5 == 0:
            print("Buzz")
        elif i / 3 == 0:
            print("FizzBizz")
        elif i / 5 == 0:
            print("Buzz")
        else:
            print(i)

# Call the function with the desired range, for example:
fizz_buzz(5)
